src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/MathHelpers.cs

```diff
@@ -2,381 +2,313 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.Diagnostics;
 using System.Runtime;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-using Internal.Runtime;
-
 namespace Internal.Runtime.CompilerHelpers
 {
     /// <summary>
-    /// Math helpers for generated code. The helpers marked with [RuntimeExport] and the type
-    /// itself need to be public because they constitute a public contract with the .NET Native toolchain.
+    /// Math helpers for generated code. The helpers here are referenced by the runtime.
     /// </summary>
+    [StackTraceHidden]
     internal static partial class MathHelpers
     {
+        private const double IntMaxValueOffset = 2147483648.0; // 2^31, int.MaxValue + 1
```

Comment:
nit: Convention is to use the .NET names, not the C# keyword names

This should be `Int32MaxValueOffset` and the next shoul dbe `UInt32MaxValueOffset`. It's generally good to follow the convention even for internal APIs.